src/components/utils/popover/components/popover-item/popover-item.ts

```diff
@@ -166,40 +138,53 @@ export class PopoverItem {
     this.disableSpecialHoverAndFocusBehavior();
   }
 
+  /**
+   * Returns list of item children
+   */
+  public get children(): PopoverItemParams[] {
+    return 'children' in this.params && this.params.children?.items !== undefined ? this.params.children.items : [];
```

Comment:
`this.params.children?.items ?? []`

Reply:
🥲
![image](https://github.com/codex-team/editor.js/assets/31101125/15baa4cb-9f21-4aa6-99dd-a7179700a47d)